layer/VkLayer_FROG_gamescope_wsi.cpp

```diff
@@ -102,6 +130,15 @@ namespace GamescopeWSILayer {
     if (appid == 1600780)
       flags |= GamescopeLayerClient::Flag::DisableHDR;
 
+    if (pApplicationInfo) {
+      // This matches regular vkd3d, not just vkd3d-proton as well...
+      // Oh well... /shrug.
+      if (pApplicationInfo->pEngineName == "vkd3d"sv ||
```

Comment:
Due to a bug discovered in https://github.com/HansKristian-Work/vkd3d-proton/pull/1892 I think vkd3d check needs to check for version as well. engineVersion > VK_MAKE_VERSION(2, 11, 1) would be needed here.